docs/index.html

```diff
@@ -193,7 +193,7 @@ <h2>Getting Involved</h2>
 <p>You can help the project tremendously by discovering and <a href="https://github.com/Leaflet/Leaflet/blob/main/CONTRIBUTING.md#reporting-bugs">reporting bugs</a>, <a href="https://github.com/Leaflet/Leaflet/blob/main/CONTRIBUTING.md#improving-documentation">improving documentation</a>,
 helping others on <a href="https://stackoverflow.com/questions/tagged/leaflet">Stack Overflow</a>, <a href="https://gis.stackexchange.com/questions/tagged/leaflet">GIS Stack Exchange</a>
 and <a href="https://github.com/Leaflet/Leaflet/issues">GitHub issues</a>,
-tweeting to <a href="https://twitter.com/LeafletJS">@LeafletJS</a>
+X to <a href="https://twitter.com/LeafletJS">@LeafletJS</a>
```

Comment:
Not quit sure what to use instead of 'tweeting', X-ing sounds kinda vulgar :laughing:

Reply:
we can use 'xeets' in place of 'tweets'....haha

Reply:
I think saying to X would be the best option.

We could also use "X's" in playce of "tweets".

Reply:
Will this issue got resolved ?
I am new to open source...when will this going to merge ?